TLM/TLM/Resources/lang_fr.txt

```diff
@@ -124,23 +124,23 @@ Unit_system Système d'unités
 Metric Système métrique
 Imperial Système impérial
 Use_more_CPU_cores_for_route_calculation_if_available Utiliser plus de cœurs du CPU pour le calcul de trajets (si disponible)
-Activated_features caractéristiques activées
-Junction_restrictions Restrictions de jonction
-Prohibit_spawning_of_pocket_cars Prohibit spawning of pocket cars
+Activated_features Fonctionnalités activées
+Junction_restrictions Restrictions des intersections
+Prohibit_spawning_of_pocket_cars Empêcher l'apparition de voitures de poche
 Reset_stuck_cims_and_vehicles Réinit. citoyens et véhicules coincés
-Default_speed_limits Limites des vitesse par défaut
+Default_speed_limits Limites de vitesse par défaut
 Looking_for_a_parking_spot Cherche une place de stationnement
 Driving_to_a_parking_spot Se dirige vers un stationnement
 Driving_to_another_parking_spot Se dirige vers un autre stationnement
 Entering_vehicle Entre dans un véhicule
 Walking_to_car Se dirige vers une voiture
 Using_public_transport Utilise les transports en commun
 Walking Marche
-Thinking_of_a_good_parking_spot Réfléchi à un bon stationnement
+Thinking_of_a_good_parking_spot Réfléchit à un bon stationnement
 Switch_view Changer de vue
 Outgoing_demand Demande sortante
 Incoming_demand Demande entrante
-Advanced_Vehicle_AI IA avancée des véhicules
+Advanced_Vehicle_AI IA des véhicules avancée
```

Comment:
Should it be `AI` or `IA`?

Reply:
In French, the acronym `IA` is used for `Intelligence artificielle`.